src/app/containers/StoryPromo/index.jsx

```diff
@@ -139,7 +139,7 @@ const StoryPromoContainer = ({
     return null;
   }
 
-  const topStory = promoType === 'top';
+  const topStory = promoType === 'top' || promoType === 'leading';
```

Comment:
We shouldn't do this, otherwise, leading promos will be using the same Image and Text styles and number of columns as top stories.
https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo/src/ImageStyles.jsx#L107
https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo/src/ImageStyles.jsx#L107